### DISCUSSION TITLE
Build takes 14mins +

### DISCUSSION THREAD

**Discussion by TheDarkStrix** [vercel/next.js#64855] [Help]
### Summary

nextjs build takes 14min+ to build , we are deploying the application via Kubernetes, the image is build via jenkins. We are also using *.modules.scss for css 

the trace file : [Click here](https://shorturl.at/qwxJQ) 

Need to understand why the long build time and how I can optimise this. Any help is greatly appreciated

**Comment by Archisketch-Finn** [comment]
I also encountered a similar issue. The deployment process with Vercel took more than 40 minutes. It consumed quite a lot of time at "Collecting page data" and "Traced Next.js server files". It took around 10 seconds to build on my personal computer.

**Comment by Archisketch-Finn** [comment]
I have also created a similar discussion and attached images of the building process, but have not received any assistance. Have you found out the reason yet? I think we should turn it into an issue.
